ptvsd/wrapper.py

```diff
@@ -461,7 +461,7 @@ def get_break_mode(self, name):
     def add_exception_break(self, exception, break_raised, break_uncaught,
                             skip_stdlib=False):
 
-        notify_on_handled_exceptions = 1 if break_raised else 0
+        notify_on_handled_exceptions = 2 if break_raised else 0
```

Comment:
Please use a constant for these magic numbers `1`, `0`, `2`
Previously i assumed these were booleans. Clearly these numbers have some special meaning.